--- /usr/lib64/python3.6/site.py.bk     2019-04-24 15:59:08.733928933 +0000
+++ /usr/lib64/python3.6/site.py        2019-04-24 15:59:26.217705346 +0000
@@ -104,11 +104,11 @@
             continue   # don't mess with a PEP 302-supplied __file__
         try:
             m.__file__ = os.path.abspath(m.__file__)
-        except (AttributeError, OSError):
+        except (AttributeError, OSError, TypeError):
             pass
         try:
             m.__cached__ = os.path.abspath(m.__cached__)
-        except (AttributeError, OSError):
+        except (AttributeError, OSError, TypeError):
             pass


